src/screens/ProfileScreen/index.js

```diff
@@ -99,12 +100,20 @@ const Header = (props) => {
         headerHeightRef.current = headerHeightLayout;
       }}>
       <View style={styles.content}>
-        <View style={{flexDirection: 'row'}}>
+        <View style={{flexDirection: 'row', alignContent: 'center', alignItems: 'center'}}>
           <ProfilePicture
             onImageContainerClick={changeImage}
             profilePicPath={dataMain.profile_pic_path}
+            karmaScore={Math.floor(dataMain.karma_score)}
           />
-          <View style={{marginLeft: 20}}>
+          <View
+            style={{
+              flexDirection: 'column',
+              paddingHorizontal: 14,
+              paddingVertical: 5,
+              justifyContent: 'center'
+            }}>
+            <KarmaScore score={Math.floor(dataMain.karma_score)} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `karmaScore` prop is being passed to the `ProfilePicture` and `KarmaScore` components. However, it's not clear if these components have been updated to accept and handle this new prop. Please ensure that these components are properly updated to handle the `karmaScore` prop.

Also, you're using `Math.floor(dataMain.karma_score)` twice. To avoid unnecessary calculations, consider storing the result in a variable and reusing it.
```diff
- karmaScore={Math.floor(dataMain.karma_score)}
+ const karmaScore = Math.floor(dataMain.karma_score);
+ ...
+ karmaScore={karmaScore}
...
- <KarmaScore score={Math.floor(dataMain.karma_score)} />
+ <KarmaScore score={karmaScore} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
needed, coz `karma_score` type is float number